beacon_api/utils/data_query.py

```diff
@@ -183,6 +210,7 @@ def handle_wildcard(sequence):
                                                     endMin_pos, endMax_pos)
                 LOG.info(f"Query for dataset(s): {datasets} that are {access_type} matching conditions.")
                 for record in list(db_response):
+                    record = add_handover(record)
```

Comment:
See comments above. Handover field should be added only if handover feature is enabled (drs is configured).

Reply:
Instead of doing `add_handover` you could add:
```
response["datasetHandover"] = make_handover(__handover_datasets__, [record['datasetId']],
                                                record['referenceName'], record['start'],
                                                record['end'], record['referenceBases'],
                                                record['alternateBases'], record['variantType'])
```
as part of `transform_misses` and `transform_record`